docs/development/blog-authoring.md

```
@@ -0,0 +1,77 @@
+# Authoring the /blog Section
+
+The blog lives at `packages/site/docs/blog/`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)